include/Server/include/ClientSession/include/ClientSession.h

```diff
@@ -0,0 +1,98 @@
+#ifndef ClientState_H
+#define ClientState_H
```

Comment:
According to the code convention, it might be better to call this header CLIENT_STATE_H.